_summaries/choromanski2020rethinking_1.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: summary
+title: Summary
+paper: choromanski2020rethinking
+# Please fill out info below
+author: joshpapermaster
+score: 8
+---
+
+<!-- TODO: Summarize the paper:
+* What is the core idea?
+* How is it realized (technically)?
+* How well does the paper perform?
+* What interesting variants are explored? -->
+
+While transformers have become a very popular way of working with sequential data, the equation to determine the attention of the model relies on softmax. This paper provides an alternative trick to approximate the attention in linear time complexity rather than the quadratic time complexity currently required. 
+
+The authors suggest a new algorithm for performers called FAVOR+, which stands for fast attention via positive orthogonal random features. 
+
+Using this approach brings the time complexity of attention from O(L^2 * D) to O(Lrd)
+- This is beneficial because we can control r as a parameter and decrease the computational effort needed.
+
+![favor+](choromanski2020rethinking_1b.png)
+
+FAVOR+ uses a feature map to represent the kernel to keep time complexity low.
+
+The similarities between the guassian kernel and the softmax kernel are noted, but guassian kernels can include negative values which leads to unstable results. 
```

Comment:
Can you add maybe - Therefore, kernel producing non-negative scores are used in this model.  
Also, can you add how they are similar or different a bit more?